gulpfile.js

```diff
@@ -60,7 +61,7 @@ function compiledCode(destination, minified, versioned) {
 		.pipe(concat(minified ? 'cmd-resume.min.js' : 'cmd-resume.js'));
 
 	stream.pipe(inject.prepend(';(function($){\n"use strict";\n\n'))
-		.pipe(inject.afterEach('\n', '  '))
+		.pipe(inject.afterEach('\n', '	'))
```

Comment:
Why did you touch this? Does it have any affect?
```suggestion
		.pipe(inject.afterEach('\n', '  '))
```

Reply:
I believe that was from me changing the tab to spaces and back to hard tab. I won't have an effect

Reply:
On correction, it has been reverted back to 2 spaces

Reply:
Best not to touch